doc/assembling.rst

```diff
@@ -29,14 +29,14 @@ In sum, equivalent to :func:`pandas.merge`, the :func:`fuzzy_join`
 has no need for pre-cleaning.
 
 
-Feature augmentation for machine learning
------------------------------------------
+Joiner for machine learning
+---------------------------
 
-Joining is pretty straigthforward for two tables: you only need to identify
+Fuzzy joining is pretty straigthforward for two tables: you only need to identify
```

Comment:
Let's start by explaining the join and mention after: oh, by the way, it's fuzzy